DataBuilders Hackathon.md

```diff
@@ -0,0 +1,88 @@
+# Gitcoin Open Data Hackathon using Ocean Protocol
+In the Ocean Protocol <> Gitcoin ODC Hackathon, contestants are incentivised to use Ocean's technology to find insightful anti-Sybil algorithms and to package anti-Sybil algorithms into composable legos with a fully decentrealised approach.
+
+
+## 0. Key Details
+
+- Kickoff: Jan 5th, 2023
+- Submission deadline: Tuesday Jan 31st, 2023 at 23:59 UTC
+- Total Prize pool: $40,000 in bounties
+
+**Outline of this README**
+
+This readme describes 2 different ways that participants can use to get free bonus points during the hackathon:
+
+1. Publish your algorithms as ERC721 using the [Ocean Market](https://market.oceanprotocol.com/publish/1) frontend
+2. Use Ocean [Data NFTs](https://docs.oceanprotocol.com/core-concepts/datanft-and-datatoken#what-is-a-data-nft) as [ERC725Y](https://github.com/ERC725Alliance/erc725/blob/main/docs/ERC-725.md) via [Ocean.py](https://github.com/oceanprotocol/ocean.py) or [Ocean.js](https://github.com/oceanprotocol/ocean.js)
+
+## 1. Use Ocean Market frontend
+
+Ocean Market provides a convenient interface for individuals and organizations to publish their data. Data assets can be datasets, algorithms,images, location information, audio, video, sales data, or combinations of all!
+
+To publish on the Ocean Marketplace, you must first get MATIC or ETH to pay for gas fees, and host your assets on a decentralized storage like Arweave.
+
+To host your data assets on Arweave you have two options:
+
+1. In a webapp, using [ardrive.io](https://www.ardrive.io). [Ocean Docs](https://docs.oceanprotocol.com/using-ocean-market/asset-hosting#arweave) has more info.
+2. In code, using [pybundlr library](https://github.com/oceanprotocol/pybundlr).
+
+Once the above is completed, take the transaction ID and head over to [Ocean Market](https://market.oceanprotocol.com/publish/1) to get started with publishing. If you need more information feel free to ask for support in our dedicated [Discord channel](https://discord.gg/JK4rq7KBGh) or visit our [docs](https://docs.oceanprotocol.com/using-ocean-market/marketplace-publish-data-asset).
+
+
+## 2. Use Ocean Data NFTs as ERC725Y
+
+The ERC725y feature enables the NFT owner to input and update information in a key-value store. These values can be viewed externally by anyone and are highly useful building blocks for various Sybil protection approaches. As an example, DAOs can use Ocean data NFTs to provide great amount of resistance in preventing misuse of DAO proposal mechanisms. 
+
+Ocean data NFTs can also be used to [Login with Web3](https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/profile-nfts-flow.md) and allow a dApp to not only connect to the user's wallet, but also access profile data that the user has privately shared to it. In fact, Ocean data NFTs can essentially be used as [Soulbound Tokens](https://papers.ssrn.com/sol3/Delivery.cfm/SSRN_ID4105763_code1186331.pdf?abstractid=4105763&mirid=1) as well. 
```

Comment:
> In fact, Ocean data NFTs can essentially be used as [Soulbound Tokens]

I'm interested to know how they can be used as soulbound tokens? I thought the whole idea of soulbound tokens is that they are non-transferable. We would surely need a new ERC721 template that removed any transfer function in order to stop people transferring their data NFTs?

Reply:
Ocean data nft template already supports the argument of transferable y/n.

The ocean.py readme for profile data nfts uses it. You can think of profile data nfts as an example of SBT.

https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/profile-nfts-flow.md

Reply:
Ok cool, sounds good

Reply:
It might be worth adding a bit more of an explanation of that here: 

```suggestion
Ocean data NFTs can also be used to [Login with Web3](https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/profile-nfts-flow.md) and allow a dApp to not only connect to the user's wallet, but also access profile data that the user has privately shared to it. Additionally, the Ocean data NFT template supports the argument of `transferable=False` which means they can be used as [Soulbound Tokens](https://papers.ssrn.com/sol3/Delivery.cfm/SSRN_ID4105763_code1186331.pdf?abstractid=4105763&mirid=1) as well. 
```